pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -1828,8 +1858,10 @@ type BackupStatus struct {
 	BackupSizeReadable string `json:"backupSizeReadable,omitempty"`
 	// BackupSize is the data size of the backup.
 	BackupSize int64 `json:"backupSize,omitempty"`
-	// CommitTs is the snapshot time point of tidb cluster.
+	// CommitTs is the commit ts of the backup, snapshot ts for full backup or start ts for log backup.
 	CommitTs string `json:"commitTs,omitempty"`
+	// TruncateUntil is log backup truncate until timestamp.
+	TruncateUntil string `json:"truncateUntil,omitempty"`
```

Comment:
What's the difference between `spec.TruncateUntil` and `status.TruncateUntil`?

Reply:
`spec.TruncateUntil` is user specified truncate until ts, `status.TruncateUntil` is current truncate until ts which is used for `get`

Reply:
`status.TruncateUntil` will be updated periodly to match the newest ts?

Reply:
`status.TruncateUntil` will be updated after truncate log backup. the proess like that:
1. we find `spec.TruncateUntil` changed
2. create job and it will start a pod and execte `./br log truncate --until=xxx`
3. after truncate, update `status.TruncateUntil`
`status.TruncateUntil` will be the value of the last time we truncate log backup if user get status before truncate finish.